notification-manager/src/main/java/com/qpay/notificationmanager/config/MailMessageGenerator.java

```diff
@@ -0,0 +1,23 @@
+package com.qpay.notificationmanager.config;
+
+import com.qpay.libs.models.PaymentNotification;
+import lombok.RequiredArgsConstructor;
+import org.springframework.mail.SimpleMailMessage;
+import org.springframework.stereotype.Component;
+
+@Component
+@RequiredArgsConstructor
+public class MailMessageGenerator {
```

Comment:
where is interface ? You can have different implementations for the same interface. MailMessageGenerator, TelegramMessageGenerator and so on. Also interfaces are better for testing purposes